modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -202,6 +204,26 @@ public Error OpenInExternalEditor(Script script, int line, int col)
                 case ExternalEditorId.None:
                     // Not an error. Tells the caller to fallback to the global external editor settings or the built-in editor.
                     return Error.Unavailable;
+                case ExternalEditorId.CustomEditor:
+                {
+                    string file = ProjectSettings.GlobalizePath(script.ResourcePath);
+                    var execCommand = _editorSettings.GetSetting(Settings.ExecPath).As<string>(); 
+                    var execArgs = _editorSettings.GetSetting(Settings.ExecArgs).As<string>(); 
+
+                    //var args = new List<string>(); 
+                    var args = execArgs.Split(","); 
+
+                    for(int i = 0; i < args.Length; i++)
+                    {
+                        args[i] = args[i].Replace("{file}", file); 
+                        args[i] = args[i].Replace("{line}", line.ToString());
+                        args[i] = args[i].Replace("{col}", col.ToString());      
+                    }
+
+                    OS.RunProcess(execCommand, args); 
```

Comment:
Should replicate / reuse the logic from https://github.com/godotengine/godot/blob/5f2137a05040beb84d14ca6dd465688d3917a3ee/editor/plugins/script_editor_plugin.cpp#L2233-L2293 (If that is really not possible, even specifying args as an array would be better than having comma separated args)

Reply:
I changed it to work as shown in this file